docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,267 @@
+
+### Add colorfilter component
+
+1. Click the **Components** subtab and click **Create component**.
+
+1. Then, select the `local modular resource` type from the list.
+
+   {{<imgproc src="extend/modular-resources/configure/add-local-module-list.png" resize="300x" declaredimensions=true alt="The add a component modal showing the list of components to add with 'local modular resource' shown at the bottom">}}
+
+1. On the next screen:
+
+   - Select the select camera from the drop down menu.
+   - Enter the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet">}} of your modular resource's [model](/extend/modular-resources/key-concepts/#models), `example:camera:colorfilter`.
+   - Enter a name for this instance of your modular resource.
+     This name must be different from the module name.
+
+   {{<imgproc src="/tutorials/pet-photographer/add-colorfilter-module-create.png" resize="400x" declaredimensions=true alt="The add a component model showing the create a module step for a local color filter module">}}
+
+1. Click **Create** to create the modular resource component.
+
+1. Copy the following JSON configuration into the Attributes section:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "vision_service": "my_color_detector",
+  "actual_cam": "cam"
+}
+```
+
+![A component panel for a color filter modular resource with the attributes filled out for vision service and actual_cam](/tutorials/pet-photographer/colorfiltercam-component-attributes.png)
+
+### Configure data capture
+
+To add data capture for the colorfilter camera, click **Add Method** in the **Data Capture configuration** section of your color filter camera component.
+Toggle the **Type** dropdown menu, select **ReadImage**, and set the **Frequency** of the capture to `0.1`.
+Then, click **Save config**.
+
+![A component panel for a color filter modular resource with the attributes filled out for vision service and actual_cam as well as the data capture configuration capture set capture ReadImage at 0.1 frequency](/tutorials/pet-photographer/colorfiltercam-component.png)
+
+### Test your color filter camera
+
+To test that your color filter camera is capturing and filtering images properly, navigate to the **Control** tab on your robot's page.
+
+On the **colorfiltercam**'s panel, toggle **view colorfiltercam** to view your camera's live feed.
+Test the filter by moving a blue colored item within the camera's field of view.
+Then, go to the **Data** tab to view pictures that contain the blue colored item.
+
+![Filtered data tab contents from colorfiltercam showing only photos of dog with blue collar](/tutorials/pet-photographer/data-capture.png)
+
+## Photograph your pet
+
+To photograph your own pet, put them in a blue collar and position your camera to point at an area they frequently visit.
+When you check the **Data** tab, you'll only see pictures of your adorable pet.
+
+## Next steps
```

Comment:
I assume people will have followed along so we don't need this here but can pick up some of it in the next steps section:
```suggestion
## Next steps

Your pet photographer is now set up, place it in an area your pet frequently visits and don't forget to attach the colored object to your pet.
Then, check the [**Data** tab](/manage/data/view/), you'll only see pictures of your adorable pet.
```